packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/agreements/16-transports-routiers/store/__tests__/store.test.ts

```diff
@@ -0,0 +1,27 @@
+import { loadPublicodesRules } from "../../../../../api";
+import { createIndemniteLicenciementStore } from "../../../../store";
+
+describe("CC 16 store", () => {
+  let store: ReturnType<typeof createIndemniteLicenciementStore>;
+
+  beforeEach(() => {
```

Comment:
un beforeAll suffirait si tu veux gagner un peu en perf sur le test. Ça éviterai de recréer le store avant chaque "it".

Reply:
Ouais mais le beforeEach permet de réinitialiser le store à 0

Reply:
Donc vaut mieux un `beforeEach`